setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup
+
+with open('README.rst') as f:
+    readme = f.read()
+
+with open('LICENSE') as f:
+    license = f.read()
+
+setup(
+    name='ensembl_srv',
+    version='3.1.0',
+    description='Server libraries for Ensembl Production infrastructure services',
+    long_description=readme,
+    author='Dan Staines',
```

Comment:
WE should think of adding our names in here :-)

Reply:
I'm okay with it, but before adding that I'd ask @thomasmaurel @james-monkeyshines

Reply:
We should add all our names, actually.